guardian/admin.py

```diff
@@ -193,8 +193,8 @@ def obj_perms_manage_view(self, request, object_pk):
                 )
                 return redirect(url)
         elif request.method == 'POST' and 'submit_manage_group' in request.POST:
-            user_form = UserManage()
-            group_form = GroupManage(request.POST)
+            user_form = self.get_obj_perms_user_select_form(request)()
+            group_form = self.get_obj_perms_group_select_form(request)(request.POST)
```

Comment:
Here you didn't modify the ``user_form`` constructor call to pass the ``request.POST`` parameter. Guessing that this should be consistent with the above change?